packages/space-opera/editor/index.html

```diff
@@ -86,50 +103,57 @@
   }
 
   me-tabs {
-    background-color: #202124;  /* GOOGLE_GREY_900 */
+    background-color: var(--expandable-section-background);
     padding: 0;
-    width: 360px;
+    width: 440px;
+    border-left: 1px solid var(--tabs-conatiner-border)
   }
 
   me-tabs a {
     color: #dadcff; /* GOOGLE_GREY_300, but.. bluer */
   }
 
   me-tabbed-panel {
-    max-height: calc(100vh - 96px);
+    max-height: calc(100vh - 45px);
     overflow-y: auto;
     display: block;
   }
 </style>
 <body>
-  <model-editor>
-    <div class="app">
+<model-editor>
+  <div class="app">
     <div class="editor-body-root">
       <div class="mvContainer">
         <model-viewer-preview id="editing_adapter">
         </model-viewer-preview>
       </div>
       <me-tabs>
-        <me-tabbed-panel icon="save">
-          <model-viewer-snippet></model-viewer-snippet>
+        <me-tabbed-panel icon="home">
+          <home-container></home-container>
+        </me-tabbed-panel>
+        <me-tabbed-panel icon="import_export">
+          <me-export-panel></me-export-panel>
```

Comment:
Let's put the snippet at the top of the export panel to be consistent with the two following panels.

Reply:
Also, let's avoid the word "Upload" in all of our UX; it implies we're taking their model to our server, which we are not. "Open" is better, possibly "Import".

Reply:
done

Reply:
<img width="347" alt="Screen Shot 2020-11-18 at 5 37 32 PM" src="https://user-images.githubusercontent.com/1649964/99609975-bf513b00-29c5-11eb-900a-8281280151a5.png">